### ISSUE TITLE
more realistic hypocalcemia

### ISSUE THREAD

**Issue by mlangsdorf** [CleverRaven/Cataclysm-DDA#24312]
According to effects.json, the effects of hypocalcemia (lack of calcium in the diet) are brittle bones.
In my life, the effects of of my hypocalcemia are charlie horses, other painful muscle spasms, tingling sensations in my extremities, and in severe enough cases that I don't intend to ever experience personally, a painful death via heart attack. (I take medicine, it's no big deal.)

see https://en.wikipedia.org/wiki/Hypocalcaemia#Signs_and_symptoms

double checking a few things before I write a PR:
1. I need to edit effects.json to add base_mod and scaling_mod entries to hypocalcaemia, with some kind of pain_min, pain_max, pain_chance values. 
2. effects can't chain to another effect via JSON - if I want severe hypocalcemia to cause the shakes effect, I have to add something in the src to make that happen.
3. hypocalcemia currently causes weak bones (a real but long term effect) instead of crippling pain (the real short term effect) because of a research oversight, and not a design decision. I know the vitamins subsystem is not completely implemented but I'm unsure if adding the actual effects of hypocalcemia is desirable.

**Comment by mlangsdorf** [comment]
I've gone mildly hypocalcemic a couple of times after I first acquired the condition that causes my hypocalcemia. It was a charlie horse once a day, not constant pain that prevented me from walking. If you go hypocalcemic enough, the pain might become crippling, but you are highly motivated to go find a solution to the problem well before you get to that level.

I think large amounts of pain reduce focus, so crafting will become impossible automatically eventually. I don't intend to override the game in that regard.

I will probably aid calcium injections as a rare drop in hospitals, since that's the recommended solution to severe acute hypocalcemia.

**Comment by FulcrumA** [comment]
Probably would use not only injections but further balancing of nutrient system as currently in-game, maintaining more varied and healthier diet that one does IRL it's hard to stave off most deficiencies, while IRL it's hardly an issue for average person, even ones not particularly caring about healthy eating or following any diet.

**Comment by Amariithynar** [comment]
Yeah, a lot of foods have a lot more nutrients than are listed, just not in massive volumes.
